docs/modules/glob.md

```diff
@@ -0,0 +1,37 @@
+# Globbing
+
+Many glob libraries are bloated and have lighter and faster alternatives.
+
+# Alternatives
+
+## fdir
+
+`fdir` is a very fast directory crawler that can achieve the same goals, despite having
+a vastly different API. If you don't use user-provided glob patterns, you probably don't
+even need to use globs with this package. It can be combined with `picomatch` if you do need them.
+
+[Project Page](https://github.com/thecodrr/fdir)
+
+[npm](https://npmjs.com/package/fdir)
+
+## Node.js (since v22.x)
+
+Node.js v22 and up supports the [`fs.glob`](https://nodejs.org/api/fs.html#fspromisesglobpattern-options)
+function. Although it's (currently) experimental, it works as a zero-dependency solution.

```

Comment:
i probably wouldn't mention that it is experimental as that'll change over time

i think it'll be enough to just mention it exists as long as we're clear about the version constraint